packages/wrangler/src/__tests__/cfetch-utils.test.ts

```diff
@@ -1,6 +1,7 @@
 import { rest } from "msw";
 import { hasMorePages } from "../cfetch";
 import { mockAccountId, mockApiToken } from "./helpers/mock-account-id";
+import { mockConsoleMethods } from "./helpers/mock-console";
 import { createFetchResult, msw } from "./helpers/msw";
 import { runInTempDir } from "./helpers/run-in-tmp";
```

Comment:
The cleanup for this util is failing on windows again (resource lock) – I thought we solved that? (maybe by just wrapping in try..catch but that seemed fine)

Reply:
I've rerun the job anyway in case it was just a flake